datasets/wiki40b/wiki40b.py

```diff
@@ -20,9 +20,9 @@
 import logging
 import os
 
```

Comment:
Now this looks like we have different versions of isort :-/

Reply:
I missed this bug so much